README.md

```diff
@@ -8,25 +8,25 @@ Installation docs are [here][operator-sdk-installation].
 To run locally, you must set the operand image as shown below.
 
 ```
-export OPERAND_IMAGE="quay.io/app-sre/updateservice:2873c6b" 
+export OPERAND_IMAGE="quay.io/app-sre/update-service:2873c6b"
```

Comment:
I should have left the image name as cincinnati when I did the renaming. I think it best to change it back to `quay.io/app-sre/cincinnati:2873c6b` to match `hack/deploy.sh` until we know what the updated OSUS image will be to use as a default.

Reply:
I agree.

Reply:
fixed https://github.com/openshift/cincinnati-operator/commit/2f41139e45ed957fcb2062cf8e7a7b64df53f61b